exercise-core/.classpath

```diff
@@ -22,5 +22,6 @@
 			<attribute name="maven.pomderived" value="true"/>
 		</attributes>
 	</classpathentry>
+	<classpathentry kind="con" path="org.eclipse.jdt.junit.JUNIT_CONTAINER/4"/>
```

Comment:
This must not be there: JUnit is taken from the Maven dependencies. This was used in the very first lessons when we were not using Maven